service/start.go

```diff
@@ -135,9 +136,11 @@ func (d *Dependency) extractVolumes() ([]container.Mount, error) {
 	return volumes, nil
 }
 
+// volumeKey create an key for the volume based on the alias to make sure that the volume
+// will be consistent from one service's version to another.
 func volumeKey(s *Service, dependency string, volume string) string {
 	return xstructhash.Hash([]string{
-		s.ID,
+		s.Alias,
```

Comment:
@antho1404 I thought alias was optional, but here we're using it all the time.

Reply:
It should be optional for the user but not internally